src/screens/Settings/Settings.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import { Text, ScrollView, KeyboardAvoidingView, View } from "react-native";
+import { Input, Button } from "react-native-elements";
+import Icon from "react-native-vector-icons/Ionicons";
+import { styles } from "./styles";
+import { translate } from "../../i18n";
+
+const Settings = () => {
+  return (
+    <ScrollView
+      contentContainerStyle={styles.scrollContainer}
+      showsVerticalScrollIndicator={false}
+      enabled
+    >
+      <KeyboardAvoidingView style={styles.container} enabled>
+        <Text style={styles.headerText}>
+          {translate("settings.profileSettings")}
+        </Text>
+
+        <Text style={styles.hintText}>{translate("settings.email")}</Text>
+        <View style={styles.inputContainer}>
+          <Input
+            placeholder=""
+            label={translate("settings.email")}
+            labelStyle={styles.inputLabel}
+          />
+        </View>
+        <View style={styles.buttonContainer}>
```

Comment:
We have 3 refresh buttons. Could we move it into a separate function?

Reply:
fixed it!